src/gui/qgsrubberband.cpp

```diff
@@ -665,7 +665,8 @@ int QgsRubberBand::numberOfVertices() const
   {
     for ( const QgsPolylineXY &ring : poly )
     {
-      count += ring.size();
+      if ( ring )
```

Comment:
A QgsPolylineXY reference can't be false here -- a different fix is needed

Reply:
mmm right.